backend/src/feeds/feed.service.ts

```diff
@@ -14,10 +13,15 @@ export class FeedService {
         this.logger = new Logger('FeedService');
     }
 
-    async getFeeds(type: string, limit: number) {
+    async getFeeds(type: string, limit: number, feedId: number) {
```

Comment:
Your `getFeeds` function does not accept params `skip` that was required from `backend/src/feeds/feed.controller.ts::getFeeds`

Reply:
it was a naming difference, feedId was used as skip.
we will get only previous items before the item till which we need to skip.